cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go

```diff
@@ -24,7 +24,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	kubeletconfigv1alpha1 "k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig/v1alpha1"
+	kubeletconfigv1beta1 "k8s.io/kubernetes/pkg/kubelet/apis/kubelet.config.k8s.io/v1beta1"
```

Comment:
Have we agreed on the convention to use the fully-qualified API group names as directory names as well?
Many other API groups have a shorthand name, which would be `kubeletconfig` in this case.

I'm fine with this, but then we should be consistent with this kind of naming for the other componentconfigs (like the proxy)

Reply:
> Have we agreed on the convention to use the fully-qualified API group names as directory names as well?

I definitely prefer this. Originally, we didn't do this because of deficiencies in the generators

Reply:
Ah, I thought we were switching directory names too. I went back and read the email thread where we discussed this and didn't find anything about directory names, so I guess it's still an open question. I would, however, still like the directory names to mirror what you see in the object for the sake of consistency.

Reply:
Saw your below comment that imports won't be consistent with the package name... so I guess we can't have our cake and eat it too. I'll probably revert the change to the directory name in light of this.

Reply:
Added a commit that changes things back to kubeletconfig